blueocean-personalization/src/main/js/components/FavoritePipeline.jsx

```diff
@@ -73,6 +74,11 @@ export class FavoritePipeline extends Component {
     }
 
     render() {
+        // TODO: this should probably key off a more generic capability like 'FavoritableItem'
+        if (capable(this.props.pipeline, 'hudson.matrix.MatrixProject')) {
+            return null;
+        }
+
```

Comment:
yes, perhaps... there are a fairly finite number of job types (the community has been good at keeping a lid on that) so I wouldn't get too excited about being generic with them.